server/.env.example

```diff
@@ -93,3 +98,29 @@ LOW_BATTERY_ALERT_TIMEOUT_TEST = 2
 # Login to our OneSignal Account. Get this value from Settings --> Messaging --> Android Categories --> Alerts and Reminders --> Channel ID
 ONESIGNAL_ALERT_ANDROID_CATEGORY_ID=guid-here
 ONESIGNAL_ALERT_ANDROID_CATEGORY_ID_TEST=android-group-guid
+
+# Name of the ClickUp Team where the Sensor Tracker lives
+# Get this from Clickup --> Settings --> Workspace Settings
+CLICKUP_TEAM_NAME=Our Team Name
+CLICKUP_TEAM_NAME_TEST=Our Team Name
+
+# ID of the ClickUp Team where the Sensor Tracker lives
+# Get this from Clickup --> In the URL https://app.clickup.com/<this number here>/home
+CLICKUP_TEAM_ID=123
+CLICKUP_TEAM_ID_TEST=123
+
+# Primary and secondary API keys used by PA to send API calls to the Sensors server
+# Two keys are used to allow for seamless API key rotation
+# Get this from Digital Ocean --> Apps --> particle-accelerator --> Settings --> Components: particle-accelerator --> Environment Variables --> REACT_APP_BRAVE_API_KEY
+PA_API_KEY_PRIMARY=MyLongKey
+PA_API_KEY_PRIMARY_TEST=MyLongKey
+PA_API_KEY_SECONDARY=MyLongKeyBackup
+PA_API_KEY_SECONDARY_TEST=MyLongKeyBackup
+
+# Primary and Secondary passwords used by PA before making any calls that will affect the DB
+# Two passwords are used to allow for seamless password rotation
+# Get this from 1Password --> Brave Sensor Credentials --> PA Dashboard Renamer password
+PA_PASSWORD_PRIMARY=Apassword
+PA_PASSWORD_PRIMARY_TEST=Apassword
+PA_PASSWORD_SECONDARY=ABpassword
+PA_PASSWORD_SECONDARY_TEST=ABpassword
```

Comment:
I'm unsure about the general style of a .env.example file, but should there be any instructions here on what to change in the frontend code when these keys/passwords get rotated?